services/web-app/pages/assets/components.module.scss

```diff
@@ -0,0 +1,63 @@
+//
```

Comment:
The design spec specifies truncation after two lines. Can you make sure the page header holds up with really long headings? I think you'll want to use the `line-clamp` mixin.